src/actions/deploy_actions.js

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright: Ambrosus Technologies GmbH
+Email: tech@ambrosus.com
+
+This Source Code Form is subject to the terms of the Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+This Source Code Form is “Incompatible With Secondary Licenses”, as defined by the Mozilla Public License, v. 2.0.
+*/
+
+import contractJsons, {contractSuperSpeedJsons} from '../contract_jsons';
+
+/**
+ * Computed as:
+ * floor((Total AMB supply) * 2% / (blocks/year))
+ */
+const DEFAULT_BLOCK_REWARD = '1146237436580416032';
```

Comment:
According to my calculations it should be: `1146237435108134836`. 

```suggestion
const DEFAULT_BLOCK_REWARD = '1146237435108134836';
```